metabrainz/views.py

```diff
@@ -21,12 +21,12 @@ def about():
 @index_bp.route('/projects')
 def projects():
     return render_template('index/projects.html')
-    
+
 
 @index_bp.route('/team')
 def team():
     return render_template('index/team.html')
-    
+
```

Comment:
These spaces didn’t belong here so it was right to remove them, but that has nothing to do with the redirect issue. Therefore, these indentation fixes should be in a separate commit.